load-data-from-kafka/main.bal

```diff
@@ -0,0 +1,44 @@
+import ballerinax/kafka;
+import ballerina/time;
+import load_data_from_kafka.leads;
+import ballerina/uuid;
+
+public type LeadAnalyticsData record {|
+    string LeadID;
+    string FirstName;
+    string LastName;
+    string Email;
+    string Phone;
+    string Company;
+    string Status;
+    string Source;
+    float LeadScore;
```

Comment:
Do we need this Lead prefix, shall we just use, `id, score, owner, createdDate, contactDate`.

Reply:
updated.